tests/backup/backup_test.go

```diff
@@ -6408,6 +6408,136 @@ var _ = Describe("{IssueMultipleDeletesForSharedBackup}", func() {
+	var bkpNamespaces []string
+	var clusterUid string
+	var cloudCredName string
+	var backupName string
+	var schBackupName string
+	var schPolicyUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	backupLocationMap := make(map[string]string)
+	timeStamp := strconv.Itoa(int(time.Now().Unix()))
+	periodicPolicyName := fmt.Sprintf("%s-%s", "periodic", timeStamp)
+	bkpNamespaces = make([]string, 0)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScheduleBackupCreation", "Create schedule backup creation with a single namespace", nil, 0)
+		log.InfoD("Application installation")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Schedule Backup Creation", func() {
+		Step("Validate deployed applications", func() {
+			ValidateApplications(contexts)
+		})
+
+		Step("Adding Cloud Account", func() {
+			log.InfoD("Adding cloud account")
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudCredName = fmt.Sprintf("%s-%s-%v", "cred", provider, timeStamp)
+				cloudCredUID = uuid.New()
+				CloudCredUIDMap[cloudCredUID] = cloudCredName
+				CreateCloudCredential(provider, cloudCredName, cloudCredUID, orgID)
+			}
+		})
+
+		Step("Adding Backup Location", func() {
+			log.InfoD("Adding Backup Location")
+			bucketNames := getBucketName()
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudCredName = fmt.Sprintf("%s-%s-%v", "cred", provider, timeStamp)
+				bucketName := fmt.Sprintf("%s-%s", provider, bucketNames[0])
+				backupLocationName = fmt.Sprintf("%s-%s", provider, bucketNames[0])
+				backupLocationUID = uuid.New()
+				backupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateBackupLocation(provider, backupLocationName, backupLocationUID, cloudCredName, cloudCredUID,
+					bucketName, orgID, "")
+				dash.VerifyFatal(err, nil, "Creating backup location")
+			}
+		})
+
+		Step("Creating Schedule Policies", func() {
+			log.InfoD("Creating schedule policy with type Periodic")
+			periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 2)
+			periodicPolicyStatus := Inst().Backup.BackupSchedulePolicy(periodicPolicyName, uuid.New(), orgID, periodicSchedulePolicyInfo)
+			dash.VerifyFatal(periodicPolicyStatus, nil, "Creating periodic schedule policy")
+		})
+
+		Step("Adding Clusters for backup", func() {
+			ctx, _ := backup.GetAdminCtxFromSecret()
+			CreateSourceAndDestClusters(orgID, "", "", ctx)
+			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			log.InfoD(clusterUid)
```

Comment:
Add a string fmt.Sprintf("Cluster UUID is %s", clusterUid)

Reply:
As this is not imp, removed same.